index.js

```diff
@@ -1,155 +0,0 @@
-    while ((higherOrLower.toUpperCase().trim() != "H") && (higherOrLower.toUpperCase().trim() != "L")) {
-    higherOrLower = await ask(`Is your number higher or lower than ${guess}? Enter H if your number is higher or L if your number is lower.\n`)
-
-    if (higherOrLower.trim().toUpperCase() === 'H') {
-      //catch a cheater! eventually the cheater will be caught when they must say that their number is above the max that was set: 
-      if (guess == max) {
-        priorGuess = guess+1
-        console.log(`You said your secret number was lower than ${priorGuess}. It can't be lower than ${lastGuess} and higher than ${guess}. There is no room for liars here. Goodbye.\n`)
-        process.exit(); 
-      } 
-      //since the secret number is higher than the guess, new minimum is guess+1, calculate new guess and update the count:  
-      else 
-        min = guess + 1; 
-        guesser(max, min)
-        counter();
-    } 
-
-    else if (higherOrLower.trim().toUpperCase() === 'L') {
-      if (guess == min) {
-        priorGuess = guess-1;
-        //catch a cheater! eventually the cheater will be caught if they say that their number is below the min: 
-        console.log(`You said your secret number was higher than ${priorGuess}. It can't be lower than ${priorGuess} and higher than ${guess}.  There is no room for liars here. Goodbye.`)
-        process.exit(); 
-      }
-      //since the secret number is lower than the guess, new max is guess-1, calculate teh new guess and update the count:  
-      max = guess -1;
-      guesser(max, min)
-      counter(); 
-    }
-  }
-  }
-}
-
-//reverse game!
-async function inverse() { 
-  //opening message 
-  await ask("Now let's play a game where I choose a number and then YOU try to guess it. Press any key when you're ready to start!\n");
-
-  //ask the max 
-  let max = await ask("What do you want the maximum in our guessing game to be?\n")
-  max = +max; 
-
-  //let the computer choose a random number as the secret number 
-  let playerGuess = 0;
-  let computerNumber = Math.round((max+1)*Math.random()); 
-  let numbersGuessed = [];
-  let count = 0; 
-  function counter () {
-    numbersGuessed.push(playerGuess)
-    count = numbersGuessed.length
-    return count 
-  }; 
-
-
-// continue asking for guess until player's guess matches computer's guess 
-  while (playerGuess !== computerNumber) {
-  playerGuess = await ask(`I'm thinking of a number between 1 and ${max}. What is your guess? \n`);
-  console.log("Not quite! Try again.")
-  playerGuess = +playerGuess; 
-  counter();
-  }
-// when playerGuess === computerNumber, exit while loop, give option to  return to main menu if they want to play another game. 
-  let playAgain = await ask(`You did it! You guessed my number in ${count} tries. \nThat was F U N. Do you want to play another game? Type YES if you do.\n`);
-  if (playAgain.toUpperCase().trim() === "YES") { 
-    games(); 
-  }
-  else process.exit(); 
```

Comment:
I found a "one off" error in here where it's possible to have the computer guess be a 0, and while it's a 0 and the way you establish your initial `playerGuess`, it can cause the game to immediately say the player wins and quit out. You can avoid the game saying you win right away by changing your line 130 to `let playerGuess;` (so that it's established but has no value to trigger the win condition. This wouldn't fix your "one off" error that's allowing the computer to pick 0 as it's random number. That would probably be something in the math on line 131